pkg/github/github-client.go

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
A small nitpick: all files in the project are using snake_case, but github-client and github-helper are using dashes. Not sure if there's a pattern I'm missing here.

Reply:
You are right, thank you for pointing out. I've renamed the files.